modules/openapi-generator/src/main/resources/ruby-client/gitlab-ci.mustache

```diff
@@ -0,0 +1,26 @@
+.ruby: &ruby
+  variables:
+    LANG: "C.UTF-8"
+  before_script:
+    - ruby -v
+    - bundle config set --local deployment true
```

Comment:
This is required to mitigate [CVE-2022-29176](https://github.com/rubygems/rubygems.org/security/advisories/GHSA-hccv-rwq6-vh79)